app/views/invoices/_form.html.erb

```diff
@@ -27,10 +27,14 @@
 
   <tr>
     <td class="subheading"><%= f.label :status, "Status" %>:</td>
-    <% if can? :status, f.object %>
+    <% if current_member.is_at_least? :tracker_management %>
```

Comment:
Use abilities; don't hard-code in member tiers because it makes it harder to find/manage abilities across the whole site when changing things later. Instead of setting member groups here you can just give more people the `:status` ability.

This is also inconsistent with the permissions you have set in `ability.rb`. This will show/hide a form but under the hood you have given all of exec all permissions on invoices regardless.